charts/astronomer/templates/houston/cronjobs/houston-populate-daily-task-metrics.yaml

```diff
@@ -0,0 +1,54 @@
+##############################################
+## Houston Populate Daily Task Metrics CronJob
+##############################################
+{{- if .Values.global.taskUsageMetricsEnabled }}
+apiVersion: {{ include "apiVersion.batch.cronjob" . }}
+kind: CronJob
+metadata:
+  name: "{{ .Release.Name }}-houston-populate-daily-task-metrics"
+  labels:
+    tier: astronomer
+    component: houston-populate-daily-task-metrics
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  schedule: {{ .Values.houston.populateDailyTaskMetrics.schedule }}
```

Comment:
this should point to  {{ .Values.global.taskUsageMetricsEnabled }}

Reply:
schedule is configured separately?